lib/db.js

```diff
@@ -0,0 +1,150 @@
+const { fetch } = require('fetch-ponyfill')();
+const FormData = require('form-data');
+const qs = require('querystring');
+const flat = require('flat');
+
+const DB_OPTIONS = require('./dbopts');
+const { httpMessage, httpBody } = require('./response-transforms');
+
+const dbOptions = (conn, options, body) => {
```

Comment:
My argument for destructuring is that it self-documents which options are needed. Also would make things a little cleaner to be able to use shorthand within the functions.